onnxscript/function_libs/torch_lib/ops/nn.py

```diff
@@ -2298,7 +2298,7 @@ def aten_upsample_bilinear2d(
     output_size: Optional[INT64] = None,
     scales_h: Optional[float] = None,
     scales_w: Optional[float] = None,
-    align_corners: bool = True,
+    align_corners: bool = False,
```

Comment:
Looks like align_corners should be required in the signature

Reply:
I don't know why it was written in this way. Maybe something to do with onnxscript signature syntax?

Reply:
I don't think it is constrained by onnxscript. Looks like something to be fixed

Reply:
Hmm, looks like tests start to fail with the default value swap.

Reply:
The issue is also we are using the tests for `torch.nn.Upsample`. We should probably take the test cases out to create tests for the aten op like @xiaowuhu suggested.

Reply:
Hmm ok, so it appears the reference test case is for [torch.nn.functional.upsample_bilinear](https://pytorch.org/docs/stable/generated/torch.nn.functional.upsample_bilinear.html#torch.nn.functional.upsample_bilinear), which is deprecated and defaults `align_corners` to True.

I agree the best fix is to make `align_corners` required, and fix the input wrangler.